templates/main.html

```diff
@@ -13,27 +13,7 @@
 
   <body>
     <div class="container">
-      <h1>Integrating a Jupyter Notebook in a Web Page</h1>
-      <p>Some text.</p>
-
-      <h2>Maths</h2>
-
-      <p>Here is a simple inline equation: $E=mc^2$.</p>
-
-      <p>And for longer equations, use the standard math mode:</p>
-      
-      $$
-      \mathcal{L}=\bar{\psi}\left(i\gamma^{\mu}\partial_  {\mu}-m\right)\psi 
-      $$
-      
-      <h2>A code section</h2>
-      <p>Here is some code:</p>
-      <pre>
-import foo
-print foo.bar()
-      </pre>
-
-      {% include 'overfitting.html' %}
+      {% include FNAME %}
```

Comment:
If used out of your scripts, FNAME is undefined. So the current behavior is not preserved. Could you use jinja2 in python instead of sed in a shell script to replace FNAME?

Reply:
Right: Here, I was using FNAME as a placeholder and replacing it in the bash Script: The "template/main.html" file was copied into "template/[name-of-the-notebook].html" file, and replaced FNAME with the name of the html created in the previous step.